### PR TITLE
Add support for proxy lists

### PR DESCRIPTION
**Pull request by Jo-Byr**
Support proxy lists in the same way as string/number lists

### CODE REVIEW

File: trame_simput/module/protocol.py [hunk 1]
```diff
@@ -33,6 +33,12 @@ def reset_cache(self):
         logger.info("reset_cache")
         self.net_cache_domains = {}
 
+    @exportRpc("simput.create_proxy")
+    def test(self, manager_id, proxy_type):
```
Comment (jourdain) [inline trame_simput/module/protocol.py:37]: should use a different method name

**Comment by jourdain** [comment]
You need to format your commit appropriately so that semantic release can trigger a new release, update the change log and tag it on github for you.

**Comment by jourdain** [comment]
Looks reasonable otherwise.

**Comment by jourdain** [comment]
Please rework/reword your commit for semantic release

**Comment by bourdaisj** [comment]
@Jo-Byr can you add an example?  
Also your change introduces a difference between the vue2/vue3 implementation. can you add a markdown file in the docs folder to quickly explain the difference between the two implementations? something like vue2_vs_vue3 or else

**Comment by Jo-Byr** [comment]
> your change introduces a difference between the vue2/vue3 implementation. can you add a markdown file in the docs folder to quickly explain the difference between the two implementations? something like vue2_vs_vue3 or else

I only implemented the proxy list support in the vue2 part of trame-simput. Maybe should I convert this PR to a draft PR for the time being and once the vue2 changes are validated I can add them to the vue3 part ?

**Comment by jourdain** [comment]
Should we merge?

**Review by bourdaisj — APPROVED** [review]
Seems reasonable

**Pull request merged**